crowdloan/kusama.json

```diff
@@ -484,5 +484,14 @@
     "description": "Unknown parachain",
     "website": "https://parachains.info",
     "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/gradient/Westend_Testnet.svg"
+  },
+  {
+    "paraid": "2226",
+    "name": "Genshiro 2",
+    "token": "GENS",
+    "description": "The boldest DeFi one-stop-shop and R&D hub",
+    "website": "https://genshiro.equilibrium.io/en",
+    "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/gradient/Genshiro.svg",
+    "rewardRate": 2000
```

Comment:
Are we have some insights about reward rate?
looks like [here](https://parachains.info/details/genshiro/) is different information

Reply:
I will remove it as it is not quiet clear what reward rate is